stories/topics/con-saas-limited-support-status.adoc

```diff
@@ -9,7 +9,8 @@ A deployment can return to a fully-supported status if you remediate the violati
 A deployment might move to a Limited Support status for many reasons, including the following scenarios:
 
 If you do not implement an execution plane:: A customer execution plane is required for automation to be performed. 
-If you have not yet configured an execution plane, or that have an execution plane in a degraded state, you will be asked to correct issues with the execution plane prior to support for automation being provided.
+If you have not yet configured an execution plane, or have an execution plane in a degraded state, you must correct issues with the execution plane prior to support for automation being provided.
 If you do not maintain the execution plane with supported dependencies:: 
-Both Red{nbsp}Hat Enterprise Linux (RHEL) and OpenShift-based execution planes will require maintenance and upgrades to ensure that those resources maintain the minimum supported versions for {PlatformNameShort} dependencies. Upgrading these resources can be achieved through different models; Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples.
-Ensuring that the OS, cluster, and receptor resources are up-to-date with supported {PlatformNameShort} versions will reduce a support dependency impact.
+Both Red{nbsp}Hat Enterprise Linux (RHEL) and OpenShift-based execution planes require maintenance and upgrades to ensure that those resources maintain the minimum supported versions for {PlatformNameShort} dependencies.
+Upgrading these resources can be achieved through different models; Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples.
```

Comment:
Maybe use active voice here if you can.
And can you rephrase "Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples."